src/deck/drivers/src/emergency_land.c

```diff
@@ -0,0 +1,103 @@
+/*
+This code controls the emergency landing option. The ermergency land option is controlled via the parameter named 'deck.eland'.
+If the param equals to 1 then the emergency landing code is launched. 
+The param is changed whenever the crazyflie receives a high digitalRead from the gpio pin 1 which is connected to the NRF's emergency land 
+output pin. Whenever the NRF sends a high signal at DECK_GPIO_IO1 then the code sets the param 'deck.eland' to 1 which causes it to launch the 
+emergency landing operation.
+*/
+#include "deck.h"
+#include "FreeRTOS.h"
+#include "system.h"
+#include "task.h"
+#include "pm.h"
+#include "debug.h"
+#include "param.h"
+
+#define TASK_SIZE configMINIMAL_STACK_SIZE
+#define EMERGENCY_LAND_CHECK_FREQUENCY 10
+
+#define TASK_PRIORITY 3
+#define EMERGENCY_LAND_PIN DECK_GPIO_IO2 //The STM pin used to receive the signal from the NRF
+
+static uint8_t isEmergencyLandInit = false;
+
+static uint8_t emergencyLandingParam = false;
+
+static paramVarId_t emergencyLandingFlagParamId = {0};
+
+static void listenToEmergencyLand(void* data) 
```

Comment:
1. add "Task" at the end of function name
2. change the parameter name `data` to `ignored`
3. add comment explaining the function